docs/how-to-guides/platform-integration/vtex/intelligent-search-reports.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Configure your store to produce Intelligent Search reports"
+sidebar_label: "Configure your store to produce Intelligent Search reports"
+---
+
+# How to configure your store to produce Intelligent Search reports
+
+This guide covers how to have Intelligent Search (IS) reports for your store. You'll be able to do this by sending the appropriate user events to IS.
+
+Intelligent Search reports might be empty because there is no user data to feed them. To do that, events have to be sent containing additional metadata returned during search queries.
+
+> Be mindful of the reference Pull Request present in the Additional Resources section. It contains all the changes required to make the Next.js starter work appropriately and might serve as an accompanying, more practical guide.
+
+---
+
+## Before you start
+
+Before you begin, you should already have:
+
+- `@faststore/api` installed with version `1.12.35` or higher. If you are coming from a previous version, be aware of possible breaking changes that might have been introduced in other versions of the package.
+
+You should also consider that:
+
+- This guide uses the Next.js starter.
+- Intelligent Search reports only consider full-text searches.
+- Reports might not immediately fill up after all the steps are done.
+
+---
+
+## Step by step
+
+### Step 1 - Adding required fields to full-text search queries
+
+1. Find every full-text search query in the store
+
+In the Next JS starters, they are the `ProductGalleryQuery` and `SearchSuggestionsQuery`.
+
+> Not every search query is full-text. You can consider a search query as being full-text if it uses the query variable `term` is not empty. 2. Add the metadata fragment to above one of the queries
```

Comment:
This phrase ended up confusing, also the 2. Should be outside of this quote right?

```suggestion
> Not every search query is full-text. You can consider a search query as being full-text if the query variable `term` is not empty. 

2. Add the metadata fragment to above one of the queries
```